packages/components/src/navigator/navigator-provider/README.md

```diff
@@ -47,6 +47,10 @@ For example:
 -   `/parent/:param` is a child of `/parent` as well.
 -   if the current screen has a `path` with value `/parent/child/grand-child`, when going "back" `Navigator` will try to recursively navigate the path hierarchy until a matching screen (or the root `/`) is found.
 
+### Height and animations
+
+Due to how `NavigatorScreen` animations work, it is recommended that the `NavigatorProvider` component is given enough height to match the tallest `NavigatorScreen`. Not doing so could result in glitchy animations, especially when transitioning from a taller to a shorter `NavigatorScreen`.
```

Comment:
Ah, this is a bummer. Looks like something the component should be able to do on its own. Is there a reason why we can't automate this, using a resize observer for example?

Reply:
I pushed some more work that should greatly improve this aspect.